mkdocs.yml

```diff
@@ -474,6 +474,7 @@ nav:
                   - Power Port: "user-guide/core-data-model/dcim/powerport.md"
                   - Power Outlet: "user-guide/core-data-model/dcim/poweroutlet.md"
                   - Rear Port: "user-guide/core-data-model/dcim/rearport.md"
+              - Module Family: "user-guide/core-data-model/dcim/modulefamily.md"
```

Comment:
Might make logical sense for this to appear before `Module Type` instead? Similar to how `Device Family` is before `Device Type`.

Reply:
agreed!